stix_shifter_modules/azure_log_analytics/stix_translation/json/to_stix_map.json

```diff
@@ -1,362 +1,1266 @@
-        "object": "alert"
+    {
+      "key": "process.binary_ref",
+      "object": "parent1",
+      "references": "file1"
     },
-    "Entities": {
-        "key": "x-azure-security-alert.entities",
-        "object": "alert"
+    {
+      "key": "process.parent_ref",
+      "object": "process",
+      "references": "parent1"
     },
-    "DisplayName": {
-        "key": "x-azure-security-alert.display_name",
-        "object": "alert"
+    {
+      "key": "x-oca-event.parent_process_ref",
+      "object": "event",
+      "references": "parent1"
+    }
+  ],
+  "ProcessId": {
+    "key": "process.pid",
+    "object": "parent1",
+    "transformer": "HexToInteger"
+  },
+  "ProcessName": [
+    {
+      "key": "directory.path",
+      "object": "file_dir1",
+      "transformer": "ToDirectoryPath"
     },
-    "WorkspaceResourceGroup": {
-        "key": "x-azure-security-alert.workspace_resource_group",
-        "object": "alert"
+    {
+      "key": "file.name",
+      "object": "file1",
+      "transformer": "ToFileName"
     },
-    "CompromisedEntity": {
-        "key": "x-azure-security-alert.compromised_entity",
-        "object": "alert"
+    {
+      "key": "file.parent_directory_ref",
+      "object": "file1",
+      "references": "file_dir1"
     },
-    "eventId": {
-        "key": "x-azure-security-event.event_id",
-        "object": "msevent"
+    {
+      "key": "process.binary_ref",
+      "object": "parent1",
+      "references": "file1"
+    }
+  ],
+  "TokenElevationType": {
+    "key": "process.x_token_elevation_type",
+    "object": "process"
+  },
+  "MandatoryLabel": {
+    "key": "process.x_mandatory_label",
+    "object": "process"
+  },
+  "FilePath": [
+    {
+      "key": "file.path",
+      "object": "file"
     },
-    "Title": {
-        "key": "x-azure-security-event.title",
-        "object": "msevent"
+    {
+      "key": "x-oca-event.file_ref",
+      "object": "event",
+      "references": "file"
     },
-    "DeviceDescription": {
-        "key": "x-azure-security-event.text",
-        "object": "msevent"
+    {
+      "key": "process.binary_ref",
+      "object": "process",
+      "references": "file"
     },
-    "DeviceId": {
-        "key": "x-oca-asset.device_id",
-        "object": "asset"
+    {
+      "key": "file.name",
+      "object": "file",
+      "transformer": "ToFileName"
     },
-    "SourceSystem": {
-        "key": "x-azure-security-event.source",
-        "object": "msevent"
+    {
+      "key": "x-ibm-finding.ioc_refs",
+      "object": "finding",
+      "references": ["file"],
+      "group": true
+    }
+  ],
+  "Fqbn": {
+    "key": "file.x_fqbn",
+    "object": "file"
+  },
+  "SHA256": {
+    "key": "file.hashes.SHA-256",
+    "object": "file"
+  },
+  "SHA1": {
+    "key": "file.hashes.SHA-1",
+    "object": "file"
+  },
+  "MD5": {
+    "key": "file.hashes.MD5",
+    "object": "file"
+  },
+  "ServiceFileName": {
+    "key": "x-oca-event.x_service_file_ref",
```

Comment:
avoid using `_ref` suffixes unless the property contains a reference object. basically use a different name or remove _ref if  the attribute doesn't have `"references": ""`

Reply:
> avoid using `_ref` suffixes unless the property contains a reference object. basically use a different name or remove _ref if the attribute doesn't have `"references": ""`

Updated